### PR TITLE
General style adjustments

### PR DESCRIPTION
**Pull request by spicymaya**
1. Added general styles to the app, masonry to adjust layout
2. Cleaned up unused dependencies
3. Refactored api.js to return custom promise in order to display error messages to the user

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Veggie Tales</title>
+    <link rel="stylesheet" type="text/css" href="/node_modules/bootstrap/dist/css/bootstrap.min.css"></link>
```
Comment (dialupdev) [inline src/index.html:8]: Nice. 🏰

File: src/js/components/FoodCard.jsx [hunk 2]
```diff
@@ -7,49 +7,70 @@ import {
...
+    };
+
+    const imagesLoadedOptions = { background: ".cardStyle" };
+
     return (
-      <Col sm="4">
+      <Masonry
+        className={"my-gallery-class"} // default ''
+        elementType={"ul"} // default 'div'
+        options={masonryOptions} // default {}
```
Comment (dialupdev) [inline src/js/components/FoodCard.jsx:38]: 👍🏻

File: src/js/components/App.jsx [hunk 3]
```diff
@@ -18,36 +19,38 @@ class App extends React.Component {
     this.state = {
       isOpen: false
     };
+    Navbar.propTypes = {
```
Comment (dialupdev) [inline src/js/components/App.jsx:22]: Nice! Never used this before. Looks pretty cool.

File: src/js/components/SingleFood.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// sf means single food
```
Comment (dialupdev) [inline src/js/components/SingleFood.scss:1]: Just a small note - if it means single food, you might as well call it `.single-food`. No need to use shorter names. :)

File: src/js/components/FoodWrapper.jsx [hunk 5]
```diff
@@ -29,6 +37,30 @@ class FoodWrapper extends React.Component {
...
+        <Route
+          exact
+          path="/"
+          render={() => (
+            <div>
+              <Card>
+                <CardImg
+                  top
+                  width="100%"
+                  src="https://66.media.tumblr.com/tumblr_lufh7n4Hp61qhujw5o1_400.gif"
```
Comment (dialupdev) [inline src/js/components/FoodWrapper.jsx:49]: Good one.

File: src/index.js [hunk 6]
```diff
@@ -1,7 +1,6 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import "./scss/app.scss";
-import "bootstrap";
```
Comment (dialupdev) [inline src/index.js:4]: Nice cleanup. 🧹

**Review by dialupdev — APPROVED** [review]
This looks really good. The commits are broken up really nicely! Esp like the FoodForm refactor being split into its own commit. Nice job.

File: src/js/components/SingleFood.scss (hunk 4 above)
Comment (spicymaya) [reply src/js/components/SingleFood.scss:1]: haha. Jamie rubbed off on me. I'll change moving forward.

**Pull request merged**